### PR TITLE
remove version as request parameter

### PR DESCRIPTION
**Pull request by djstrong**

### CODE REVIEW

File: packages/sdk/src/index.ts [hunk 1]
```diff
@@ -657,7 +657,7 @@ class NameGuard {
 
     const network = options?.network || this.network;
 
-    const url = `${this.endpoint}/${this.version}/inspect-namehash/${network}/${namehash}`;
+    const url = `${this.endpoint}/inspect-namehash/${network}/${namehash}`;
```
[the remark this answers is not in the document]
Comment (djstrong) [reply packages/sdk/src/index.ts:660]: I don't know why it works without `/` ;)

[the remark this answers is not in the document]
Comment (notrab) [reply packages/sdk/src/index.ts:660]: Since we use `new URL` it will take care of that.

**Review by notrab — APPROVED** [review]
lgtm: I pushed some changes to the SDK that removes the `version` property. We can easily add this back later if required. I will need to publish a new version of the SDK once the API is deployed.

File: packages/sdk/src/index.ts [hunk 2]
```diff
@@ -433,14 +433,12 @@ class NameGuardError extends Error {
 }
 
 const DEFAULT_ENDPOINT = "https://api.nameguard.io";
-export const API_VERSION = "v0.8-beta";
```
Comment (djstrong) [inline packages/sdk/src/index.ts:436]: @notrab Doesn't it needs to stay? Isn't this imported by ReportFooter?

File: packages/sdk/src/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (djstrong) [reply packages/sdk/src/index.ts:652]: @notrab So how it should look everywhere? `${this.endpoint}/inspect-namehash` or `${this.endpoint}inspect-namehash`?

[the remark this answers is not in the document]
Comment (djstrong) [reply packages/sdk/src/index.ts:652]: It is not working properly:
image
image

**Review by notrab — APPROVED** [review]

**Pull request merged**